README.md

```diff
@@ -73,26 +88,54 @@ pip install lightning-thunder
   <summary>Advanced install options</summary>
     <!-- following section will be skipped from PyPI description -->
 
+&#160;
+
 ### Install from main
 
+Alternatively, you can install the latest version of Thunder directly from this GitHub repository as follows:
```

Comment:
Note that the nvfuser and pytorch nightly steps are required regardless of pip vs github vs local clone. We should clarify that

Reply:
Thanks. I updated this and added this as explicit step.